lib/dialogs/promo/connected_wallet_promo_dialog.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:ax_dapp/service/dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class ConnectedWalletPromoDialog extends StatelessWidget {
+  const ConnectedWalletPromoDialog({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final _height = MediaQuery.of(context).size.height;
+    final _width = MediaQuery.of(context).size.width;
+    var wid = 500.0;
+    const edge = 40.0;
+    if (_width < 505) wid = _width;
+    var hgt = 345.0;
+    if (_height < 350) hgt = _height;
```

Comment:
nit: We should really use a theme for all these styles dimensions, colors, etc… otherwise we keep on having to repeat the same code over and over again through the app. https://docs.flutter.dev/cookbook/design/themes